README.md

```diff
@@ -145,6 +147,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Segonatech](https://segonatech.com/) | Battisputali, Kathmandu | Web Development, Mobile Development, Digital Marketing and Branding, ERP Solutions
 - [Semantic Creation](http://www.semanticcreation.com/) | Pulchowk, Lalitpur | Provide professional services covering all areas of modern technology
 - [Smartmobe Solutions](https://smartmobe.com/) | Tripureshwor, Kathmandu | Mobile Apps, Graphic Design, and Web Development Services
+- [Softhover](https://www.softhover.com/) | Bansbari Rd & Kapan Marg, Kathmandu | Mobile Apps and Web Development Services
```

Comment:
@bhattaraib58 Can you shorten the description down to `Bansbari Rd, Kathmandu` or `Kapan Marg, Kathmandu`?

Reply:
sure dai

Reply:
Sure dai.